### PR TITLE
Chapter 7

### PR DESCRIPTION
**Pull request by nuriman1**

### CODE REVIEW

**Comment by nuriman1** [comment]
ready

**Comment by nuriman1** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (saobv-2223) [inline app/assets/stylesheets/users.scss]: remove file that never used

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,3 +1,28 @@
 class UsersController < ApplicationController
   def new; end
+  
+  def show
+    @user = User.find_by id: params[:id]
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb:5]: add case that user is not found

File: app/assets/stylesheets/users.scss (hunk 1 above)
Comment (nuriman1) [reply app/assets/stylesheets/users.scss]: done

**Comment by nuriman1** [comment]
ready

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb:5]:
```suggestion
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:danger] = "Some Messages"
    redirect_to some_path
```

**Comment by nuriman1** [comment]
ready

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (nuriman1) [reply app/controllers/users_controller.rb:5]: done

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (nuriman1) [reply app/controllers/users_controller.rb:5]: done

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**